### PR TITLE
Updated RU lacale

### PR DESCRIPTION
**Pull request by Whitebrim**

### CODE REVIEW

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 1]
```diff
@@ -1,30 +1,48 @@
...
-  "gui.automodpack.screen.loading.finish": "Почти закончил...",
-
-  "gui.automodpack.screen.danger.title": "Внимание! Модпак найден!",
-  "gui.automodpack.screen.danger.description": "Вы хотите скачать обновление для модпака?",
-  "gui.automodpack.screen.danger.secDescription": "Может быть рисковано! Не скачивайте если не доверяете админам сервера!",
-  "gui.automodpack.screen.danger.button.accept": "Скачать",
-  "gui.automodpack.screen.danger.button.cancel": "Не скачивать"
-}
+	"automodpack.restart.full": "Модпак был скачан!",
+	"automodpack.restart.update": "Обновление Модпака было скачано!",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:3]:
```suggestion
	"automodpack.restart.update": "Обновление модпака было скачано!",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.restart.description": "Чтобы завершить обновление Модпака нужно перезагрузить игру.",
+	"automodpack.restart.secDescription": "Вы хотите перезагрузить ее прямо сейчас?",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:5]: **restart**: _перезапустить_
**reload**: _перезагрузить_

Restart usually implies full shutdown and subsequent startup while the reload updates process state in-place.

Also, punctuation.

```suggestion
	"automodpack.restart.description": "Чтобы завершить обновление модпака, нужно перезапустить игру.",
	"automodpack.restart.secDescription": "Вы хотите сделать перезапуск сейчас?",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 3]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.restart.confirm": "Перезагрузить",
+	"automodpack.restart.cancel": "Нет, обратно в меню",
+
+	"automodpack.delete": "Удалить Модпак",
+	"automodpack.delete.description": "Вы потеряете все скачанные файлы из Модпака.",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:10]:
```suggestion
	"automodpack.delete.description": "Вы потеряете все файлы из загруженного модпака.",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.delete.secDescription": "Чтобы завершить удаление Модпака нужно перезагрузить игру.",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:11]: Punctuation.

```suggestion
	"automodpack.delete.secDescription": "Чтобы завершить удаление модпака, нужно перезапустить игру.",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 5]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.delete.thiDescription": "Вы точно хотите удалить Модпак?",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:12]:
```suggestion
	"automodpack.delete.thiDescription": "Вы точно хотите удалить модпак?",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 6]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.delete.confirm": "Удалить и выйти",
+	"automodpack.delete.cancel": "Назад в меню",
+
+	"automodpack.danger": "Этот сервер предоставляет Модпак!",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:16]:
```suggestion
	"automodpack.danger": "Этот сервер предоставляет модпак!",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 7]
```diff
@@ -1,30 +1,48 @@
...
+	"automodpack.changelog.view": "Посмотреть список изменений",
+	"automodpack.changelog.noChanges": "Изменения не найдены.",
+
+	"automodpack.download.calculating": "Обработка...",
+	"automodpack.download.eta": "Времени осталось: %s",
+	"automodpack.download.downloading": "Скачивание:",
+	"automodpack.download.noFiles": "Нет файлов для скачивания!",
+
+	"automodpack.redownload": "Перекачать Модпак",
+	"automodpack.redownload.description": "Это может исправить проблемы с Модпаком.",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:31]:
```suggestion
	"automodpack.redownload.description": "Это может исправить проблемы с модпаком.",
```

File: src/main/resources/assets/automodpack/lang/ru_ru.json [hunk 8]
```diff
@@ -1,30 +1,48 @@
...
+
+	"automodpack.error": "Что-то пошло не так!",
+	"automodpack.error.files": "Не удалось скачать некоторые файлы.",
+	"automodpack.error.critical": "Критическая ошибка при скачивании Модпака.",
```
Comment (tie) [inline src/main/resources/assets/automodpack/lang/ru_ru.json:35]:
```suggestion
	"automodpack.error.critical": "Критическая ошибка при скачивании модпака.",
```

**Review by tie — CHANGES_REQUESTED** [review]
LGTM, minor wording improvements and punctuation fixes would be nice though.

File: src/main/resources/assets/automodpack/lang/ru_ru.json (hunk 1 above)
Comment (Whitebrim) [reply src/main/resources/assets/automodpack/lang/ru_ru.json:3]: In en_us Modpack is spelled with capital `M`

**Comment by Whitebrim** [comment]
@tie

**Comment by Skidamek** [comment]
thanks guys

**Pull request merged**